mongo/integration/unified/schema_version.go

```diff
@@ -16,8 +16,7 @@ import (
 
 var (
 	supportedSchemaVersions = map[int]string{
-		// We do not fully support the 1.5 schema, but we need 1.5 to test with observeSensitiveCommands.
-		1: "1.5",
```

Comment:
I believe we now fully support the 1.5 schema. I added this comment with support for `observeSensitiveCommands` when we had not yet implemented support for `serverless: forbid`. I think we can continue on to schema version 1.6 now.